internal/npm_package/npm_package.bzl

```diff
@@ -129,6 +130,11 @@ NPM_PACKAGE_ATTRS = {
         doc = """Files inside this directory which are simply copied into the package.""",
         allow_files = True,
     ),
+    "node_context_data": attr.label(
```

Comment:
If this is internal should it be `_` prefixed?

Reply:
in the rollup case, I override from a test. I guess I have no such test right now for npm_package doing the stamp thing but I like them being the same, and we ought to have such a test

Reply:
ah, the test for npm_package stamping was just busted. when I fixed the test it required passing this data to override --stamp. Fixed